Sources/Sentry/SentryWatchdogTerminationTrackingIntegration.m

```diff
@@ -64,12 +65,15 @@ - (BOOL)installWithOptions:(SentryOptions *)options
         [[SentryWatchdogTerminationLogic alloc] initWithOptions:options
                                                    crashAdapter:crashWrapper
                                                 appStateManager:appStateManager];
+    SentryScopeContextPersistentStore *scopeContextStore =
+        [[SentryScopeContextPersistentStore alloc] initWithFileManager:fileManager];
```

Comment:
Why aren't we using the instance managed by `SentryDependencyContainer` here, that is injected into init the SDC-managed `SentryWatchdogTerminationContextProcessor`? Doesn't that mean we'll have two instances competing for access to the same `contextFileURL`?

Reply:
Since `SentryWatchdogTerminationTracker` has a different dispatch queue wrapper instance (`io.sentry.watchdog-termination-tracker`) than  the one in `SentryWatchdogTerminationContextProcessor` (`io.sentry.watchdog-termination-tracking.context-processor`), and none of the operations in `SentryScopeContextPersistentStore` are thread safe, there could be issues here.